src/heroic.rs

```diff
@@ -0,0 +1,252 @@
+use std::collections::HashMap;
+
+use crate::{
+    config::RootsConfig,
+    manifest::{Manifest, Store},
+    prelude::{normalize_title, StrictPath},
+};
+
+// Deserialization of Heroic gog_store/installed.json
+#[derive(serde::Deserialize)]
+struct HeroicInstalledGame {
+    #[serde(rename = "appName")]
+    game_id: String,
```

Comment:
Adapting from camel case makes sense, but I think we should keep the name itself for consistency:

```rust
    #[serde(rename = "appName")]
    app_name: String,
```

Reply:
The reason I changed it is that `app_name` actually contains a game id.  This is consistent across all heroic and legendary json files so either name seems to be ok to me.  Shall I keep it as in the json or do you prefer a more content oriented variable name?

Reply:
I agree that "ID" fits better than "name", but since we're dealing with an external interface, I'd like to use it as-is. At least for me, it just makes it a little easier to cross-reference with the file. We could always add a doc comment:

```rust
    /// This is an opaque ID, not the human-readable title.
    #[serde(rename = "appName")]
    app_name: String,
```

Reply:
Doc comment I take then.